test/quests_test.cpp

```diff
@@ -31,16 +31,21 @@ TEST(QuestTest, SinglePlayerBadPools)
 	EXPECT_EQ(Quests[Q_SKELKING]._qactive, QUEST_NOTAVAIL) << "Skeleton King quest is deactivated with 'bad' seed";
 	ResetQuests();
 
+	// Having this seed for level 15 requires starting a game at 1977-12-28 07:44:42 PM or 2087-02-18 10:43:02 PM
+	// Given Diablo was released in 1996 and it's currently 2024 this will never have been naturally hit. It's not
+	//  possible to hit this case on a win32 system but it may be possible on macos or win64?
```

Comment:
Diablo won't allow times past 2038 so not possible on 64bit systems either. By naturally do you mean excluding setting the BIOS to 1977 on an NT based system?

Reply:
Yeah sorry, I forgot the pre-1980 date restriction was for pre-NT kernels, not specifically win32. The 2038 restriction is static linked so we can be confident that future date isn't possible. The past date could be reached using a windows release with the system date set back in time.